psycopg/psycopg/cursor.py

```diff
@@ -781,7 +781,8 @@ def stream(
             first = True
             while self._conn.wait(self._stream_fetchone_gen(first)):
                 # We know that, if we got a result, it has a single row.
-                rec: Row = self._tx.load_row(0, self._make_row)  # type: ignore
+                rec = self._tx.load_row(0, self._make_row)
+                assert rec is not None
```

Comment:
Actually I dropped this assert investigating in the stream() slowness (https://github.com/psycopg/psycopg/commit/562e8c46bbe5d2c93d4e4ac2203c7255d363c708). It wasn't caused by this assert of course, but I'm still mildly upset that Mypy checks affect runtime. I forgot to update the async side too.

What do you think? Two asserts or two type-ignore?

Reply:
Ah, didn't noticed that; `# type: ignore` are fine I think, changed accordingly.